SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
+                                        completionHandler:(void(^_Nonnull)(GADRemoteModel *
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Similarly, should we create NSDictionary parameters within each method and just request the articleId
+    //from the front end?
+    
+}
+
++ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
```

Comment:
With this method name, we want to make sure it works with both a JSON array and a JSON object. This function seems to assume that `json` will contain an array of JSON objects.